From ef45012adea2b4ff4e3c529fdb568ff2a335fd1b Mon Sep 17 00:00:00 2001
From: Vitaly Andrianov <vitalya@ti.com>
Date: Mon, 28 Aug 2017 17:04:22 -0400
Subject: [PATCH 53/54] arm: dts: rename am572x-idk-jailhouse.dts to dtsi

We need to create two more dts files for IDK with LCD support.
This commit renames jailhouse dts to dtsi, which will be included to all
idk-jailhouse dts files.

Signed-off-by: Vitaly Andrianov <vitalya@ti.com>
---
 arch/arm/boot/dts/am572x-idk-jailhouse.dts  | 106 ----------------------------
 arch/arm/boot/dts/am572x-idk-jailhouse.dtsi | 104 +++++++++++++++++++++++++++
 2 files changed, 104 insertions(+), 106 deletions(-)
 delete mode 100644 arch/arm/boot/dts/am572x-idk-jailhouse.dts
 create mode 100644 arch/arm/boot/dts/am572x-idk-jailhouse.dtsi

diff --git a/arch/arm/boot/dts/am572x-idk-jailhouse.dts b/arch/arm/boot/dts/am572x-idk-jailhouse.dts
deleted file mode 100644
index ed08fb9..0000000
--- a/arch/arm/boot/dts/am572x-idk-jailhouse.dts
+++ /dev/null
@@ -1,106 +0,0 @@
-/*
- * Copyright (C) 2017 Texas Instruments Incorporated - http://www.ti.com/
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "am572x-idk.dts"
-
-/ {
-	reserved-memory {
-		jailhouse: jailhouse@ef000000 {
-			reg = <0x0 0xef000000 0x0 0x1000000>;
-			no-map;
-			status = "okay";
-		};
-
-		jh_inmate: jh_inmate@ee000000 {
-			reg = <0x0 0xe0000000 0x0 0xf000000>;
-			no-map;
-			status = "okay";
-		};
-	};
-};
-
-&timer2 {
-	status = "disabled";
-	ti,no-idle;
-};
-
-&timer8 {
-	status = "disabled";
-	ti,no-idle;
-};
-
-&uart9 {
-	status = "disabled";
-	ti,no-idle;
-};
-
-/ {
-	ocp {
-		pruss1_eth {
-			status = "disabled";
-		};
-
-		pruss2_eth {
-			status = "disabled";
-		};
-
-		crossbar_mpu: crossbar@4a002a48 {
-			ti,irqs-skip = <10 44 127 129 133 134 135 136 137 139 140>;
-		};
-	};
-};
-
-&pruss1 {
-	status = "disabled";
-	ti,no-idle;
-};
-
-&pru1_0 {
-	status = "disabled";
-	ti,no-idle;
-};
-
-&pru1_1 {
-	status = "disabled";
-	ti,no-idle;
-};
-
-&pruss2 {
-	status = "disabled";
-	ti,no-idle;
-};
-
-&pru2_0 {
-	status = "disabled";
-	ti,no-idle;
-};
-
-&pru2_1 {
-	ti,no-idle;
-	status = "disabled";
-};
-
-&pruss1_mdio {
-	ti,no-idle;
-	status = "disabled";
-};
-
-&pruss2_mdio {
-	ti,no-idle;
-	status = "disabled";
-};
-
-&gpio5 {
-	ti,no-idle;
-	status = "disabled";
-};
-
-&gpio3 {
-	ti,no-idle;
-	status = "disabled";
-};
diff --git a/arch/arm/boot/dts/am572x-idk-jailhouse.dtsi b/arch/arm/boot/dts/am572x-idk-jailhouse.dtsi
new file mode 100644
index 0000000..727378b
--- /dev/null
+++ b/arch/arm/boot/dts/am572x-idk-jailhouse.dtsi
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2017 Texas Instruments Incorporated - http://www.ti.com/
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+/ {
+	reserved-memory {
+		jailhouse: jailhouse@ef000000 {
+			reg = <0x0 0xef000000 0x0 0x1000000>;
+			no-map;
+			status = "okay";
+		};
+
+		jh_inmate: jh_inmate@ee000000 {
+			reg = <0x0 0xe0000000 0x0 0xf000000>;
+			no-map;
+			status = "okay";
+		};
+	};
+};
+
+&timer2 {
+	status = "disabled";
+	ti,no-idle;
+};
+
+&timer8 {
+	status = "disabled";
+	ti,no-idle;
+};
+
+&uart9 {
+	status = "disabled";
+	ti,no-idle;
+};
+
+/ {
+	ocp {
+		pruss1_eth {
+			status = "disabled";
+		};
+
+		pruss2_eth {
+			status = "disabled";
+		};
+
+		crossbar_mpu: crossbar@4a002a48 {
+			ti,irqs-skip = <10 44 127 129 133 134 135 136 137 139 140>;
+		};
+	};
+};
+
+&pruss1 {
+	status = "disabled";
+	ti,no-idle;
+};
+
+&pru1_0 {
+	status = "disabled";
+	ti,no-idle;
+};
+
+&pru1_1 {
+	status = "disabled";
+	ti,no-idle;
+};
+
+&pruss2 {
+	status = "disabled";
+	ti,no-idle;
+};
+
+&pru2_0 {
+	status = "disabled";
+	ti,no-idle;
+};
+
+&pru2_1 {
+	ti,no-idle;
+	status = "disabled";
+};
+
+&pruss1_mdio {
+	ti,no-idle;
+	status = "disabled";
+};
+
+&pruss2_mdio {
+	ti,no-idle;
+	status = "disabled";
+};
+
+&gpio5 {
+	ti,no-idle;
+	status = "disabled";
+};
+
+&gpio3 {
+	ti,no-idle;
+	status = "disabled";
+};
-- 
1.9.1

